Added details to CONTRIBUTIONS.md file.

HariNithyaRao opened the pull request:
Contributed a python script please check it.

Commit:
Added details to CONTRIBUTIONS.md.

HariNithyaRao committed:
Merge pull request #1 from HariNithyaRao/HariNithyaRao

Added details to CONTRIBUTIONS.md.

Commit:
Create HariNithya.md

HariNithyaRao committed:
Merge pull request #2 from HariNithyaRao/HariNithyaRao

Create HariNithya.md

Commit:
Create asciiArtGenerator_HariNithya.py

HariNithyaRao committed:
Merge pull request #3 from HariNithyaRao/HariNithyaRao

Create asciiArtGenerator_HariNithya.py

praveensingh212223 commented:
Do not push .idea folder. Also add comments and explain what you are trying to do how this script is useful

HariNithyaRao committed:
Delete .idea directory

HariNithyaRao committed:
Update asciiArtGenerator_HariNithya.py

HariNithyaRao committed:
Update asciiArtGenerator_HariNithya.py

HariNithyaRao committed:
Update asciiArtGenerator_HariNithya.py

HariNithyaRao commented:
Hello👋, I have deleted the .idea folder and added comments to the script.


Thank you

On Thu, Oct 26, 2023 at 5:27 PM praveensingh212223 ***@***.***>
wrote:

> Do not push .idea folder. Also add comments and explain what you are
> trying to do how this script is useful
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/praveensingh212223/hacktoberfest_2023/pull/74#issuecomment-1780968941>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARSFVCSPX622EYXHTU7QBZTYBJF2HAVCNFSM6AAAAAA6Q7A2G2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBQHE3DQOJUGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

praveensingh212223 reviewed (approved):
Good, keep it up.
